exchanges/gdax.js

```diff
@@ -204,10 +213,15 @@ Trader.prototype.getOrder = function(order, callback) {
 };
 
 Trader.prototype.cancelOrder = function(order, callback) {
-  var result = function(err, data) {
-    // todo, verify result..
-    callback();
-  };
+    // callback for cancelOrder should be true if the order was already filled, otherwise false
+    var result = function(err, data) {
+        if(err) {
+            log.error('Error cancelling order:', err);
+            callback(true);  // need to catch the specific error but usually an error on cancel means it was filled
```

Comment:
If you don't return here you will also call callback(false).

Reply:
Since it was already merged can you implement this fix?

Reply:
I updated my branch with the returns...